app/src/main/java/com/devshish/internship/presentation/model/ChartItem.kt

```diff
@@ -0,0 +1,13 @@
+package com.devshish.internship.presentation.model
+
+import com.devshish.internship.domain.model.Artist
+import com.devshish.internship.domain.model.Track
+
+sealed class ChartItem {
```

Comment:
You do not need ChartItem here - you can just create 3 adapters - one for title, one for artist on for track, and then use concat adapter